pkgs/tools/networking/ssl-proxy/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, stdenv, buildGo118Module, fetchFromGitHub }:
```

Comment:
are you using stdenv anywhere? if not, can be removed

Reply:
done!